plugins/@grouparoo/mysql/__tests__/table-import/import-properties.ts

```diff
@@ -361,20 +361,26 @@ describe("mysql/table/recordProperties", () => {
           aggregationMethod: AggregationMethod.Exact,
         });
         expect(values[record.id][properties[0].id]).toEqual([
+          "Apple",
+          "Orange",
+          "Blueberry",
```

Comment:
👍 - Please update the postgres tests as well

Reply:
I wonder if it is time for `app-templates` to build this test suite for us automatically for all the SQL plugins